__tests__/__system__/cli/upload/file-to-data-set/__snapshots__/cli.upload.file-to-data-set.test.ts.snap

```diff
@@ -37,6 +37,11 @@ exports[`upload file to data set command should display upload file to data set
 
       Upload content in binary mode.
 
+   --dcb  (string)
+
+      DCB parameters for dataset allocation if not existing. It's space separated like
```

Comment:
I think this needs to be updated since the input type is an array, which means they are supposed to be comma separated, space separated (I believe) without quotes, OR repetition, `--dcb a=1,b=2` or `--dcb a=1 b=2` OR `-dcb a=1 --dcb b=2`

Reply:
@std4lqi, No need to take any action at this time though, It's accurate to say space-separated.
I just don't want people to be confused when they see the example to be comma-separated